recipe/meta.yaml

```diff
@@ -12,18 +12,19 @@ source:
 build:
    number: 0
    features:
-     - vc9      [win and py27]
-     - vc10     [win and py34]
-     - vc14     [win and py35]
+     - vc9      # [win and py27]
+     - vc10     # [win and py34]
+     - vc14     # [win and py35]
 
 requirements:
   build:
-    - gmp      [not win]
+    - gmp      # [not win]
+    # Python is listed as a build dependency purely to enumerate the different
+    # vc versions (i.e. we need one build per vc version).
+    - python   # [win]
 
   run:
-    # python is a Windows run dependency in order to get the correct vc feature
-    - python   [win]
-    - gmp      [not win]
+    - gmp      # [not win]
```

Comment:
On windows python needs to be a run requirement not a build requirement or else the package can conflict with itself